pep-0012.rst

```diff
@@ -580,6 +600,47 @@ should be avoided.  For ordinary text, use ordinary 'single-quotes' or
 above), use double-backquotes::
 
     ``literal text: in here, anything goes!``
+    
+    
+Suggested Sections
+==================
+Various sections are found to be common across PEPs and are outlined in
+PEP 1 [1]_. Those sections are provided here for convenience (sans the
+"Copyright" section and the metadata footer).
+::
```

Comment:
Again I wonder if we could put a sentence in each explaining what goes where?

Reply:
So the only reason I didn't put it in was to minimize coupling with PEP 1, but I'm fine with adding one-line guidance in each section.